ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -149,6 +150,7 @@ data PBftParams = PBftParams {
       -- but this implementation follows the specification by fixing that
       -- parameter to the ambient security parameter @k@.
     , pbftSignatureThreshold :: !Double
+    , pbftNetworkMagic       :: !NetworkMagic
```

Comment:
It's also not clear to me that this is used, except to set `protocolNetworkMagic` which also appears to be unused.